src/Documentation/glossary.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Glossary",
+  "desc": "This guide is aimed to familiarize the users with definitions to relevant DVC concepts and terminologies which are frequently used.",
+  "contents": [
+    {
+      "name": "Workspace directory",
+      "match": ["workspace"],
+      "enun": "/ˈwəːkspeɪs dɪˈrɛkt(ə)ri/",
+      "desc": "Also abbreviated as workspace, it is the root directory of a project where DVC is initialized by running `dvc init` command. Therefore, this directory will contain a `.dvc` directory as well."
+    },
+    {
+      "name": "Cache directory",
+      "match": ["cache"],
+      "enun": "/kaʃ dɪˈrɛkt(ə)ri/",
```

Comment:
That would be too much I think in our case, to include enuns.

Reply:
@shcheklein Okay, I'll remove it. Just thought it will look nice :smile: